src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
I mean, I don't like that 
```rust
|x| { x *<caret> x }
```
works.

Imagine that our block is 100 lines long, possibly with multiple nested blocks. User will encounter similar situation to the one we had with expand module.

Instead I think this inspection should be only available here:
```rust
|x| <caret>{ x * x }
```
and eventually here:
```rust
|x| { x * x }<caret>
```

See linked implementation in Kotlin how they did it there for reference. The same applies to inverse intention. 

But before implementing this I would love to hear feedback from others, espacially @matklad but he's "afk" now :smile: